discord/http.py

```diff
@@ -447,17 +504,17 @@ def send_multipart_helper(
 
     def send_files(
         self,
-        channel_id,
+        channel_id: Snowflake,
         *,
-        files,
-        content=None,
-        tts=False,
-        embed=None,
-        nonce=None,
-        allowed_mentions=None,
-        message_reference=None,
-        components=None,
-    ):
+        files: Sequence[File],
+        content: Optional[str] = None,
+        tts: bool = False,
+        embed: Optional[embed.Embed] = None,
+        nonce: Optional[str] = None,
+        allowed_mentions: Optional[message.AllowedMentions] = None,
+        message_reference: Optional[Snowflake] = None,
```

Comment:
Isn't `message_reference` a dict?
```suggestion
        message_reference: Optional[message.MessageReference] = None,
```